src/code/z_eff_spark.c

```diff
@@ -162,7 +162,7 @@ void EffectSpark_Draw(void* thisx, GraphicsContext* gfxCtx) {
         gSPSetGeometryMode(POLY_XLU_DISP++, G_ZBUFFER | G_SHADE | G_SHADING_SMOOTH);
         gDPPipeSync(POLY_XLU_DISP++);
 
-        vertices = GRAPH_ALLOC(gfxCtx, ALIGN16(this->numElements * sizeof(Vtx[4])));
+        vertices = GRAPH_ALLOC(gfxCtx, this->numElements * sizeof(Vtx[4]));
```

Comment:
So it actually doesn't match exactly as you have it, it needs to have `4 * sizeof(Vtx)` as part of 1 computation before `this->numElements` gets multiplied. So to do it like this it would have to be
`vertices = GRAPH_ALLOC(gfxCtx, this->numElements * (4 * sizeof(Vtx)));`
Is that fine?

Reply:
Seems fine to me, certainly better than current.